src/shared/Styles/GlobalStyles.tsx

```diff
@@ -49,6 +49,12 @@ export const GlobalStyles = createGlobalStyle`
   .toast-notification {
     z-index: 100000;
   }
+
+  @media screen and (min-width: 768px) {
```

Comment:
Should `@media screen and (min-width: ${(props) => props.theme.screens.smUp}) {` be used here to take advantage of the items you made in the theme file?